pkgs/applications/blockchains/oxen/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, lib, fetchurl, fetchFromGitHub, fetchpatch
+, cmake, pkgconfig
+, boost, openssl, unbound
+, pcsclite, readline, libsodium, hidapi
+, rapidjson
+, curl, sqlite
+, CoreData, IOKit, PCSC
+, trezorSupport ? true
+,   libusb1  ? null
+,   protobuf ? null
+,   python3  ? null
+}:
+
+assert stdenv.isDarwin -> IOKit != null;
+assert trezorSupport -> lib.all (x: x!=null) [ libusb1 protobuf python3 ];
+
+stdenv.mkDerivation rec {
+  pname = "oxen";
+  version = "9.1.3";
+
+  src = fetchFromGitHub {
+    owner = "oxen-io";
+    repo = "oxen-core";
+    rev = "v${version}";
+    sha256 = "11g3wqn0syk47yfcsdql5737kpad8skwdxhifn2yaz9zy8n3xqqb";
+    fetchSubmodules = true;
+  };
+
+  # Required for static linking, the only supported install path
+  lbzmqsrc = fetchurl {
```

Comment:
Maybe we should move this to an extra package that it receives update etc.

Reply:
It has to match exactly the bithash that they will use in their particular receipt for static build. I'd leave it here. They know how they want it.

Reply:
Thats a pity.